### PR TITLE
Make sure flyspell is loaded when adding the prog-mode-hook

### PR DESCRIPTION
**Pull request by steve-downey**
Make sure the flyspell package is loaded and configured if using
flyspell-prog-mode as a prog-mode-hook.

----

#

### CODE REVIEW

File: modules/init-prog-mode.el [hunk 1]
```diff
@@ -18,6 +18,12 @@
...
 (add-hook 'prog-mode-hook 'exordium-require-final-newline-mode)
 
+(use-package flyspell
+  :if (eq exordium-spell-check :prog)
+  :diminish 'flyspell-mode
+  :init
+  (add-hook 'prog-mode-hook 'flyspell-prog-mode))
+
 (when (eq exordium-spell-check :prog)
   (add-hook 'prog-mode-hook 'flyspell-prog-mode))
```
Comment (pkryger) [inline modules/init-prog-mode.el:29]: I don't think this is needed anymore
```suggestion
```

**Review by pkryger — CHANGES_REQUESTED** [review]
I think there's a redundant hook addition. Doesn't matter that much, since `add-hook` doesn't add the same thing twice, does it? Just to avoid confusion, I'd rather remove the old way.

File: modules/init-prog-mode.el (hunk 1 above)
Comment (steve-downey) [reply modules/init-prog-mode.el:29]: Yes, there's no good reason to do it twice.

**Review by pkryger — APPROVED** [review]
LGTM

**Pull request merged**